### PR TITLE
Fix #241: WIndows installation

### LINKED ISSUE #241: CouchDB not installed on Windows
When launching PASTA-ELN on Windows for the first time, CouchDB is not installed.

See the log file for the details.
[log.txt](https://github.com/PASTA-ELN/pasta-eln/files/14771001/log.txt)

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**
Fix 241
- test if .NET 3.5 is installed
- add this requirement to the documentation

### CODE REVIEW

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj Thank you for pointing out this: I updated that code and tested it on my windows partition

File: pasta_eln/installationTools.py [hunk 1]
```diff
@@ -91,6 +91,19 @@ def runAsAdminWindows(cmdLine:list[str]) -> None:
   return
 
 
+def checkForDotNetVersion() -> bool:
+  ''' check if .NET version 3.5 is installed (only on windows)
+  '''
+  import subprocess
+  command = ['reg','query','HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\NET Framework Setup\\NDP','/s']
+  result = subprocess.run(command, stdout=subprocess.PIPE, check=True)
```
[the remark this answers is not in the document]
Comment (jmurugan-fzj) [reply pasta_eln/installationTools.py:99]: So we allow only this specific version of .NET, is this intended?

b'Version    REG_SZ    3.5'

**Review by jmurugan-fzj — APPROVED** [review]

File: pasta_eln/installationTools.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SteffenBrinckmann) [reply pasta_eln/installationTools.py:99]: yes

[the remark this answers is not in the document]
Comment (jmurugan-fzj) [reply pasta_eln/installationTools.py:99]: @SteffenBrinckmann Ahh Okay, please see the failing GitHub actions!

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj The user can have multiple .net versions installed: my system has >10.

**Pull request merged**